x-pack/plugins/enterprise_search/public/applications/app_search/components/source_engines/source_engines_logic.test.ts

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { LogicMounter, mockFlashMessageHelpers, mockHttpValues } from '../../../__mocks__';
+
+jest.mock('../engine', () => ({
+  EngineLogic: { values: { engineName: 'test-engine' } },
+}));
```

Comment:
Sorry I'm a lying liar who lies, forgot one more optional comment / suggestion - we have a mock helper for this if you want to use it:

```suggestion
import { LogicMounter, mockFlashMessageHelpers, mockHttpValues } from '../../../__mocks__';
import '../../__mocks__/engine_logic.mock';
```

it does set the name to `some-engine` vs `test-engine` so you'd have to update an assertion it looks like, up to you. Feel free to skip if you'd rather, I have it as a tech debt item for myself to do a repo-wide pass for this later in any case

Reply:
Thanks for mentioning this, I addressed it in 818c7d9